onnxruntime/core/providers/qnn/builder/qnn_quant_params_wrapper.h

```diff
@@ -148,6 +169,9 @@ class QnnQuantParamsWrapper {
   // - QNN_QUANTIZATION_ENCODING_AXIS_SCALE_OFFSET: array of scale/zp pairs [{scale0, zp0}, {scale1, zp1}, ...]
   // - QNN_QUANTIZATION_ENCODING_BW_AXIS_SCALE_OFFSET: parallel arrays for scales and zps [scale0, ...] [zp0, zp1, ...]
   std::unique_ptr<char[]> per_channel_data_;
+  // - QNN_QUANTIZATION_ENCODING_BLOCK: array of scale/zp pairs [{scale0, zp0}, {scale1, zp1}, ...]
+  // - QNN_QUANTIZATION_ENCODING_BLOCKWISE_EXPANSION: array of scale/zp pairs [{scale0, zp0}, {scale1, zp1}, ...]
+  std::unique_ptr<char[]> per_block_data_;
```

Comment:
I think maybe `per_block_data_` can be combined with `per_channel_data_`. This is just an array of opaque bytes to store scales/offsets. Maybe we can call it `scale_offsets_data_`? This will reduce the size of this struct.